index.js

```diff
@@ -122,7 +122,22 @@ function parse (opts, cb) {
       file = resolve.sync(filename, { basedir: path.dirname(file) })
     }
 
-    fs.readFile(file, 'utf8', read)
+    if (opts.transformer) {
+    	var cmd = opts.transformer.split(" ") ; 
+    	var args = cmd.slice(1).map(function(a){ return a=="$$"?file:a }) ;
+    	var p = require('child_process').spawn(cmd[0],args) ;
```

Comment:
- please put all `require` calls at the top of the program
- also please name variables something descriptive